tools/travis.ps1

```diff
@@ -226,8 +228,15 @@ elseif($Stage -eq 'Build')
         Remove-Item -force ${telemetrySemaphoreFilepath}
     }
 
+    # Running tests which do not require sudo.
     $pesterPassThruObject = Start-PSPester @pesterParam
 
+    # Running tests, which require sudo.
+    pesterParam['Tag'] = @('RequireSudo')
+    pesterParam['ExcludeTag'] = @()
+    pesterParam['Sudo'] = $true
+    $pesterPassThruObject += Start-PSPester @pesterParam
```

Comment:
later you need to deal with both objects, so you cannot overwrite the first one.